### DISCUSSION TITLE
Filters are not getting honoured in clickhouse 21.11.3.6 ? Does anyone else observed it?

### DISCUSSION THREAD

**Discussion by mahi1997** [ClickHouse/ClickHouse#37663] [General]
HI 
I'm facing a very rare issue with Clickhouse version 21.11.3.6 in production.
Issue: Looks like my filters are not getting honoured. It's rare and intermittent. Has anyone else faced it?

**Comment by den-crane** [comment]
We are not aware about such issue.
21.11. is out of support. Upgrade to 22.5.1.2079
